test-setup.js

```diff
@@ -17,8 +17,13 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
 import 'core-js/stable';
 import 'regenerator-runtime/runtime';
+import { TextEncoder, TextDecoder } from 'util';
 
 process.env.ONE_CONFIG_ENV = 'test';
 
 // Required for logger tests
 process.setMaxListeners(15);
+
+// Issue with Jest this could be moved to amex-jest-preset-react
```

Comment:
what's the issue about?

Reply:
`jest-environment-jsdom` does not provide TextEncode and TextDecoder globals. This result in a `"ReferenceError: TextEncoder is not defined"` when running tests.